src/pages/RFOX/hooks/helpers.ts

```diff
@@ -31,3 +31,26 @@ export const scaleDistributionAmount = (affiliateRevenueRuneBaseUnit: bigint) =>
   // https://snapshot.org/#/shapeshiftdao.eth/proposal/0x0bb84bdf838fb90da922ce62293336bf7c0c67a9a1d6fe451ffaa29284722f9f
   return affiliateRevenueRuneBaseUnit / 4n
 }
+
+export const getRfoxContractCreationBlockNumber = (contractAddress: string) => {
+  switch (contractAddress) {
+    case '0x1094c4a99fce60e69ffe75849309408f1262d304':
+      return 222952418n
+    case '0xac2a4fd70bcd8bab0662960455c363735f0e2b56':
+      return 222913582n
```

Comment:
question: Not related to this PR but wouldn't it be better to consume these addresses from the environment variables?

We could also put them inside some constants

Reply:
We could also remove the testing contract from here altogether - we're probably roing to remove the lower cooldown period (now commented out so effectively dead code) from develop envs anyway 🙏🏽

Reply:
prefer we do this in separate task since it's unrelated